### ISSUE TITLE
[NEXT-771] Preload incorrectly added for nomodule polyfill script

### ISSUE THREAD

**Issue by timneutkens** [vercel/next.js#46793]
The nomodule polyfill scripts shouldn't add a preload as it'll cause it to be preloaded in all browsers.

Report: [https://github.com/vercel/next.js/discussions/41745#discussioncomment-5207027](https://github.com/vercel/next.js/discussions/41745#discussioncomment-5207027)

<sub>From [SyncLinear.com](https://synclinear.com) | [NEXT-771](https://linear.app/vercel/issue/NEXT-771/preload-incorrectly-added-for-nomodule-polyfill-script)</sub>

**Comment by IgorKowalczyk** [comment]
I also have the same issue

![image](https://user-images.githubusercontent.com/49127376/223247452-29ad3c6d-d435-4b56-a568-b2ed4ca8da04.png)
![image](https://user-images.githubusercontent.com/49127376/223247682-ef26f9db-4d76-4de0-ae6f-83976e1d0bf3.png)

```
    Operating System:
      Platform: linux
      Arch: x64
      Version: #1 SMP Debian 5.10.140-1 (2022-09-02)
    Binaries:
      Node: 16.15.1
      npm: 9.0.1
      Yarn: N/A
      pnpm: 7.19.0
    Relevant packages:
      next: 13.2.3
      eslint-config-next: 13.2.3
      react: 18.2.0
      react-dom: 18.2.0
```

**Comment by bwalding** [comment]
Another issue I ran into with the preload change (sometime after 12.3.0) is that there is no nonce on the `<link>` element - this caused a lot of CSP reports in Chrome (which requires a nonce IIUC). Firefox does not need a nonce on the preload links.

The site doesn't break - because the actual scripts all have a nonce - but our CSP reporting was getting hammered by the preloads failing and getting reported.  

I'm not sure if this is all part of the same issue - or whether it deserves a whole new ticket.

**Comment by JesseKoldewijn** [comment]
I'm also having this issue and though it might be fixable with a browserslist (since something similar used to work on next12) but no dice.

**Comment by ijjk** [comment]
Closing as this is fixed in https://github.com/vercel/next.js/pull/46970 which is available in `v13.2.4-canary.9` of Next.js

**Comment by JesseKoldewijn** [comment]
Awesome! Nice one!👌😁
